app/controllers/api/v1/users_controller.rb

```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
 class Api::V1::UsersController < ApplicationController
+  include Authenticable
+
+  before_action :user, only: %i[show update destroy]
```

Comment:
Why do you need this?

Reply:
It will be in ApplicationController, but is needed for the user auth, this module takes the token and checks if it corresponds to the user.

Reply:
You already have an user method in this file, so it is confusing.